### PR TITLE
Added documentation to emscoring yaml file

### PR DESCRIPTION
**Pull request by amjjbonvin**
Cleaned scripts

All tests and tox passing.

### CODE REVIEW

File: src/haddock/modules/scoring/emscoring/__init__.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ class HaddockModule(BaseCNSModule):
     name = RECIPE_PATH.name
 
     def __init__(self, order, path, initial_params=DEFAULT_CONFIG):
-        cns_script = Path(RECIPE_PATH, "cns", "scoring.cns")
+        cns_script = Path(RECIPE_PATH, "cns", "emscoring.cns")
```
Comment (joaomcteixeira) [inline src/haddock/modules/scoring/emscoring/__init__.py:23]: nice one!

**Review by joaomcteixeira — APPROVED** [review]
Perfect! Nice `git mv` and update of the `emscoring` default path!

File: src/haddock/modules/scoring/emscoring/__init__.py (hunk 1 above)
Comment (amjjbonvin) [reply src/haddock/modules/scoring/emscoring/__init__.py:23]: Yes - planning to add soon `mdscoring`

Comment (joaomcteixeira) [reply src/haddock/modules/scoring/emscoring/__init__.py:23]: :relaxed:

you can merge

**Pull request merged**